tensorboard/webapp/widgets/line_chart_v2/lib/paint_brush.ts

```diff
@@ -40,7 +38,9 @@ export class PaintBrush {
       polyline,
       paintOpt
     );
-    this.context.renderCache.setToCurrentFrame(cacheId, newCacheValue);
+    if (newCacheValue) {
```

Comment:
Looking at paint_brush.ts's signatures alone, I'd expect that `setLine(/* args for valid line */)` followed by `setLine(/* args for null line */)` to result in null or no entry in the current frame map.  With this new condition, we are saying that if `setLine` fails to create a valid line, it will preserve the existing one.

Could you help me understand when why this change is needed?